config/environments/production.rb

```diff
@@ -1,101 +1,23 @@
 require Rails.root.join("config/smtp")
 Rails.application.configure do
-  # Settings specified here will take precedence over those in
-  # config/application.rb.
-
-  # Code is not reloaded between requests.
   config.cache_classes = true
-
-  # Eager load code on boot. This eager loads most of Rails and
-  # your application in memory, allowing both threaded web servers
-  # and those relying on copy on write to perform better.
-  # Rake tasks automatically ignore this option for performance.
   config.eager_load = true
-
-  # Full error reports are disabled and caching is turned on.
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
-
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like
-  # NGINX, varnish or squid.
-  # config.action_dispatch.rack_cache = true
-
-  # Disable serving static files from the `/public` folder by default since
-  # Apache or NGINX already handles this.
-  config.serve_static_files = ENV["RAILS_SERVE_STATIC_FILES"].present?
-  config.static_cache_control = "public, max-age=#{1.year.to_i}"
-  # Enable deflate / gzip compression of controller-generated responses
+  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.